### PR TITLE
Documentation for new parameters in logstash-output-opensearch 2.0.0

### PR DESCRIPTION
**Pull request by deepdatta**
Add documentation for 3 new parameters added in logstash-output-opensearch 2.0.0

Signed-off-by: Deep Datta <deedatta@amazon.com>

### Description
Add documentation for 3 new parameters added in logstash-output-opensearch 2.0.0
https://github.com/opensearch-project/logstash-output-opensearch/releases/tag/2.0.0

### Issues Resolved
https://github.com/opensearch-project/logstash-output-opensearch/pull/174

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _clients/logstash/ship-to-opensearch.md [hunk 1]
```diff
@@ -144,6 +145,9 @@ output {
     - Instance profile credentials delivered through the Amazon EC2 metadata service
 - template (path) - You can set the path to your own template here, if you so desire. If not set, the included template will be used.
 - template_name (string, default => "logstash") - defines how the template is named inside Opensearch
+- service_name (string, default => "es") - defines the service name to be used for `aws_iam` authentication.
```
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:148]:
```suggestion
- service_name (string, default => "es") - Defines the service name to be used for `aws_iam` authentication.
```

File: _clients/logstash/ship-to-opensearch.md [hunk 2]
```diff
@@ -144,6 +145,9 @@ output {
     - Instance profile credentials delivered through the Amazon EC2 metadata service
 - template (path) - You can set the path to your own template here, if you so desire. If not set, the included template will be used.
 - template_name (string, default => "logstash") - defines how the template is named inside Opensearch
```
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:147]:
```suggestion
- template_name (string, default => "logstash") - Defines how the template is named inside Opensearch
```

**Review by Naarcha-AWS — APPROVED** [review]

File: _clients/logstash/ship-to-opensearch.md [hunk 3]
```diff
@@ -144,6 +145,9 @@ output {
     - Instance profile credentials delivered through the Amazon EC2 metadata service
 - template (path) - You can set the path to your own template here, if you so desire. If not set, the included template will be used.
```
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:146]: Remove "if you so desire".
```suggestion
- template (path) - You can set the path to your own template here. If not template is specified, the included template will be used.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _clients/logstash/ship-to-opensearch.md [hunk 4]
```diff
@@ -143,7 +143,7 @@ output {
     - Environment variables - AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY (RECOMMENDED since they are recognized by all the AWS SDKs and CLI except for .NET), or AWS_ACCESS_KEY and AWS_SECRET_KEY (only recognized by Java SDK)
     - Credential profiles file at the default location (~/.aws/credentials) shared by all AWS SDKs and the AWS CLI
     - Instance profile credentials delivered through the Amazon EC2 metadata service
-- template (path) - You can set the path to your own template here, if you so desire. If not set, the included template will be used.
+- template (path) - You can set the path to your own template here. If not template is specified, the included template will be used.
```
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:146]:
```suggestion
- template (path) - You can set the path to your own template here. If no template is specified, Logstash uses the default template.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

**Comment by Naarcha-AWS** [comment]
@deepdatta: Your PR require DCO sign-off before we can merge. For instructions on how to amend your commits, try an [interactive rebase](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md#fix-option-3-interactive-rebase).

**Comment by deepdatta** [comment]
Made the change and added Signoff

File: _clients/logstash/ship-to-opensearch.md [hunk 5]
```diff
@@ -142,8 +143,11 @@ output {
...
     - Credential profiles file at the default location (~/.aws/credentials) shared by all AWS SDKs and the AWS CLI
     - Instance profile credentials delivered through the Amazon EC2 metadata service
-- template (path) - You can set the path to your own template here, if you so desire. If not set, the included template will be used.
-- template_name (string, default => "logstash") - defines how the template is named inside Opensearch
+- template (path) - You can set the path to your own template here. If no template is specified, the plugin uses the default template.
+- template_name (string, default => "logstash") - Defines how the template is named inside Opensearch
+- service_name (string, default => "es") - Defines the service name to be used for `aws_iam` authentication.
+- legacy_template (boolean, default => true) - Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composabe templates via the _index_template API.
```
Comment (Naarcha-AWS) [inline _clients/logstash/ship-to-opensearch.md:149]: Fix spelling for "composable". 
```suggestion
- legacy_template (boolean, default => true) - Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composable templates via the _index_template API.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
One last change here.

**Review by Naarcha-AWS — APPROVED** [review]

**Review by lizsnyder — APPROVED** [review]

**Pull request merged**